students.txt

```diff
@@ -6,7 +6,9 @@ Accepted challange to read 5 book during summer.
 List of students
 
 1. Anastasia P.
-2. Natalia T.
+2. Natalie Tbilieli
+Taekwondo warrior, music lover, inspired traveler, a fan of good literature. 
+Can't imagine my life without ice cream.
```

Comment:
You forgot to add one empty line at the end of your description. Please add. And your files have a conflict, but we'll look at our meeting (aka class room)